exonum-java-binding/core/src/main/java/com/exonum/binding/core/runtime/ServiceWrapper.java

```diff
@@ -209,4 +209,9 @@ String getPublicApiRelativePath() {
     return UrlEscapers.urlPathSegmentEscaper()
         .escape(getName());
   }
+
+  Node getNode() {
```

Comment:
I'd suggest #close/requestToStop instead of exposing the Node — see comment in SR.